extensions/wikia/FeedsAndPosts/WikiVariables.class.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+class WikiVariables {
+	public function get() {
+		global $wgServer, $wgDBname, $wgDisableHTTPSDowngrade, $wgEnableHTTPSForAnons;
+
+		$wikiVariables = [
+			'basePath' => $wgServer,
+			'dbName' => $wgDBname,
+		];
+
```

Comment:
Cannot we use existing `MercuryApi::getWikiVariables` method? It has the basePath, dbName and all the rest: https://witcher.fandom.com/wikia.php?controller=MercuryApi&method=getWikiVariables

Reply:
Actually, we can. We did this to minimize the payload for feeds.